### PR TITLE
Fix/ui adapter icons

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes adapters list view
- Adds icons provided by sean
- Adds icons to plugins in adapter create view.
screen shot 2015-04-30 at 8 38 18 pm
screen shot 2015-04-30 at 8 38 23 pm

### CODE REVIEW

File: cdap-ui/app/features/adapters/services/etl-create-factory.js [hunk 1]
```diff
@@ -6,12 +6,39 @@ angular.module(PKG.name + '.feature.adapters')
...
+      return icon;
+    }
+
     AdapterApiFactory.prototype.fetchSources = function(adapterType) {
       this.dataSrc.request({
         _cdapPath: '/templates/' + adapterType + '/extensions/source'
       })
         .then(function(res) {
           this.scope.defaultSources = res;
+          if (res.length) {
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/services/etl-create-factory.js]: isn't the res.length redundant?
for forEach won't iterate if it's empty, anyways, right?

Comment (ajainarayanan) [inline cdap-ui/app/features/adapters/services/etl-create-factory.js]: Fixed.

**Comment by elfenheart** [comment]
Isn't the Stream Sink icon wrong? Should we use the wavy lines like for Streams Datasets?

**Comment by elfenheart** [comment]
Minor comment, otherwise :+1: LGTM

**Comment by elfenheart** [comment]
:+1: LGTM

**Pull request merged**